### PR TITLE
Describe this project & link to build instructions

### PR DESCRIPTION
**Pull request by ronen**
Hi guys,

Thanks for taking on maintenance of gitx!  

For the benefit of others like myself who come across this project and want to know what's up and how to use it, I figured it would be helpful for the README to be more up to date.  Here's my quick stab.  I also keep the old contents in place since I didn't want to presume too much.  Of course feel free to edit/rewrite it in ways that you feel more appropriate. 

And I also took the liberty of creating a wiki page with build instructions, scraped from @ssp's gist plus my own experience.  Again of course feel free to improve it.

Feeling like I'm taking liberties, since I'm not in a position to be an actual contributor to the code :)  But as an avid gitx user, I want to at least do my little bit to help this project along.

Cheers

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -1,5 +1,31 @@
 # What is GitX?
 
+GitX is an OS X (MacOS) Native graphical client for the `git` version control system.
+
+
+GitX has a long history of various branches and versions maintained by various people over the years.  This github org & repo are an attempt to consolidate and move forward with a current, common, community-maintained version.   See discussion at [**Forking, and plans for the future of GitX**](https://github.com/gitx/gitx.github.io/issues/1).
```
Comment (rjmunro) [inline README.markdown]: I don't think we should hard code a link to the issue into the README, I think we should distill the results of the discussion, or link to more information in a file in the project.

Minor typgraphical issues (which match the old README):
* Hard wrap to 72 chars or so (Markdown will still show it as one paragraph, but it's easier to read the source)
* Don't put two spaces after a `.`

File: README.markdown [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+<br>
+<br>
+<br>
+<br>
+<br>
+
+
+## [Old README]
+
+Below here is the contents of the README from the @rowanj fork of GitX.  Much of the information there is obsolete.
```
Comment (rjmunro) [inline README.markdown]: We should edit the old README, not leave it in. The old README will remain in the git history.

File: README.markdown (hunk 1 above)
Comment (ronen) [inline README.markdown]: Agreed.  Just trying to do the bare minimum to have something current in the README

File: README.markdown (hunk 2 above)
Comment (ronen) [inline README.markdown]: I agree here too.  But felt presumptuous to completely wipe out the README of somebody else's project.  Again just trying to do the bare minimum to have a more current (and not misleading) README

**Comment by ronen** [comment]
@rjmunro I won't be offended if you reject this PR entirely and put in something better!   My PR is really just an attempt at a quick band-aid fix to having more current than just the old obsolete README, and to include a link to build instructions.  (So IMHO one could do worse than merge the PR, then you folks who really know what you're doing could continue to improve it.)

**Comment by ronen** [comment]
I've removed the old README and did minor cleanups.  Left the link to the issue, since I'm not the right person to distill or summarize the discussion.

**Pull request merged**